src/module/functions/mod.rs

```diff
@@ -219,6 +224,9 @@ impl ModuleFunctions {
     }
 
     /// Get a parallel iterator of this module's local functions
+    ///
+    /// Requires the `parallel` feature of this crate to be enabled.
+    #[cfg(feature = "parallel")]
     pub fn par_iter_local(&self) -> impl ParallelIterator<Item = (FunctionId, &LocalFunction)> {
```

Comment:
What if instead of `cfg`ing this method into existence or not, if the parallel feature wasn't enabled, then we returned a normal iterator? I think this would have less of an impact on various call sites, and should Just Work, right? Urgh well I guess since this is a public function, and enabling the feature changes the signature, that's a no go :-/